grobid-trainer/resources/dataset/acknowledgment/evaluation/test.tei.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
If you use the n-fold validation you could move all the data in the training, which will be partitioned before running the test/evaluation sets and then you can use the whole dataset to train, with a bit more data.